### ISSUE TITLE
Dialog with black content

### ISSUE THREAD

**Issue by ThirtySomething** [codecadwallader/codemaid#451]
### Environment
- Visual Studio version: 2013 Professional Update 5
- CodeMaid version: 10.1.93 (not readable, black dialog)
- Code language: C++

### Description
All dialogs are displayed with black area - everything inside the dialog is covered by this area.

### Steps to recreate
Don't know how to reproduce this - I'm the only one in a group of three developers with this problem. But I'm also the only one with several Visual Studio installations. Currently there is
* Visual Studio 2010 Professional
* Visual Studio 2013 Professional
* Visual Studio 2015 Community
* Visual Studio 2017 Community

installed, and in each installation an instance of codemaid is installed, too.

### Current behavior
A dialog is displayed, but it's content is not visible. There is only a black area covering the complete content. No text, no controls.

### Expected behavior
Expected to see text and some controls to interact with the dialog.

**Comment by codecadwallader** [comment]
Thanks for reporting the issue.  I haven't heard that one before.  Some questions:

1.  Is it happening across all Visual Studio editions, or just VS2013?
2.  Was it previously working correctly, or has it always exhibited this behavior?
3.  CodeMaid->Options is the most obvious dialog.  What about these others:
a. CodeMaid->About
b. CodeMaid->Cleanup All Code (will bring up a confirmation dialog.  If that's black too press 'N' to select the No option).
c. CodeMaid->Spade (it's not a dialog, but it's also presenting WPF content same as the dialogs)

**Comment by ThirtySomething** [comment]
1. I've tried to call the CodeMaid about dialog in each version. The result is
* VS 2010 - Comes up and shows version 0.8.1
* VS 2013 - Comes up with black contents
* VS 2015 - Comes up and shows version 10.3
* VS 2017 - Comes up and shows version 10.3
2. I remember that it's working in the past without any problems. This was the version I've exported the settings for usage in VS 2015 and VS 2017. And as long as it's working (I see the results in the edit windows) I don't care about. But suddenly I discovered this black dialogs for files which are not part of the project. I don't have any idea what happens.
3. a. CodeMaid->About => Black frame
b. CodeMaid->Cleanup All Code => The confirmation dialog comes up as usual, but the progress dialog is displayed as a dialog with a black frame.
c. CodeMaid->Spade => This feature works only in VS 2017, in all other versions I don't see the method/function signatures. Also in the YAIP++ project, spade does not work correctly. In the file YAIP++ only the contents of namespace std are displayed. Content of the other namespace (org.derpaul.yaip) are not shown.

WPF is a good hint, I'll check the affected machine for correct WPF installation.
